packages/edit-site/src/components/navigation-sidebar/navigation-panel/menus/template-parts.js

```diff
@@ -9,17 +9,31 @@ import { map } from 'lodash';
 import { __ } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
 import {
-	__experimentalNavigationItem as NavigationItem,
 	__experimentalNavigationMenu as NavigationMenu,
+	__experimentalNavigationItem as NavigationItem,
 } from '@wordpress/components';
+import { useState, useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import TemplateNavigationItem from '../template-navigation-item';
 import { MENU_ROOT, MENU_TEMPLATE_PARTS } from '../constants';
+import SearchResults from '../search-results';
+
+const renderSearchResultItem = ( templatePart ) => (
+	<TemplateNavigationItem
+		item={ templatePart }
+		key={ `wp_template_part-${ templatePart.id }` }
+	/>
+);
```

Comment:
Any particular reasons to use a renderItem here instead of rendering `TemplateNavigationItem` directly in `SearchResults`? 🤔

Reply:
At first, I wanted this to be reusable for posts and pages search as well. But it's getting more and more complex so I'll give it another thought.